.travis.yml

```diff
@@ -43,10 +45,13 @@ after_success:
      bash ./maintainer/deploy_docs.sh
 
 env:
-   global:
-      - secure: "HIj3p+p2PV8DBVg/KGUx6n83KwB0ASE5FwOn0SMB9zxnzAqe8sapwdBQdMdq0sXB7xT1spJqRxuxOMVEVn35BNLu7bxMLfa4287C8YXcomnvmv9xruxAsjsIewnNQ80vtPVbQddBPxa4jKbqgPby5QhhAP8KANAqYe44pIV70fY="
-      - GH_DOC_BRANCH: develop
-      - GH_REPOSITORY: github.com/MDAnalysis/mdanalysis.git
-      - GIT_CI_USER: TravisCI
-      - GIT_CI_EMAIL: TravisCI@mdanalysis.org
-      - MDA_DOCDIR: package/doc/html
\ No newline at end of file
+  global:
+    - secure="HIj3p+p2PV8DBVg/KGUx6n83KwB0ASE5FwOn0SMB9zxnzAqe8sapwdBQdMdq0s  XB7xT1spJqRxuxOMVEVn35BNLu7bxMLfa4287C8YXcomnvmv9xruxAsjsIewnNQ80vtPVbQddBPxa4  jKbqgPby5QhhAP8KANAqYe44pIV70fY="
+    - GH_DOC_BRANCH=develop
+    - GH_REPOSITORY=github.com/MDAnalysis/mdanalysis.git
+    - GIT_CI_USER=TravisCI
+    - GIT_CI_EMAIL=TravisCI@mdanalysis.org
+    - MDA_DOCDIR=package/doc/html
+  matrix:
```

Comment:
I prefer if this was at the top of the file where we define the python versions we use.